src/tracer/MOM_generic_tracer.F90

```diff
@@ -612,13 +612,23 @@ subroutine MOM_generic_tracer_column_physics(h_old, h_new, ea, eb, fluxes, Hml,
         if (g_tracer_is_prog(g_tracer)) then
           do k=1,nk ;do j=jsc,jec ; do i=isc,iec
             h_work(i,j,k) = h_old(i,j,k)
+            g_tracer%boundary_forcing_tend(i,j,k) = 0
```

Comment:
Since boundary_forcing_tend is initizliaes as zero array:
https://github.com/NOAA-CEFI-Regional-Ocean-Modeling/ocean_BGC/blob/e6910057ac61ce6053a7a8cee877ef6ce858e921/generic_tracers/generic_tracer_utils.F90#L1075-L1076

Do we still need this line if the user did not request output for this term in the `diag_table`?

Reply:
Good catch, this does seem redundant